setup/config/application.config.php

```diff
@@ -5,7 +5,7 @@
  */
 
 use Magento\Setup\Mvc\Bootstrap\InitParamListener;
-use Laminas\Mvc\Service\DiAbstractServiceFactoryFactory;
+use Laminas\ServiceManager\Di\DiAbstractServiceFactoryFactory;
 use Laminas\ServiceManager\Di\DiAbstractServiceFactory;
```

Comment:
```suggestion
```

Reply:
Based on documentation for lamins-servicemanager since v.3.0.0 laminas/laminas-servicemanager-di is not required and factories map could be removed from line https://github.com/magento/magento2/blob/06d3ade50c1492c50e0dcc7e2c8371bc8b18548a/setup/config/application.config.php#L28 too